src/components/Button/styles.module.scss

```diff
@@ -0,0 +1,21 @@
+@import 'scss/global';
+
+.button {
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 40px;
+  border-radius: .8rem;
```

Comment:
I looked it over some more and I think the button could be tweaked to get rid of the odd-looking border around it.

```suggestion
  border-radius: .8rem;
  border: none;
```

|Before|After|
|---|---|
|![Capture](https://user-images.githubusercontent.com/48393820/82491182-51799480-9ab2-11ea-8417-b4ed5acacb9d.PNG)|![Capture2](https://user-images.githubusercontent.com/48393820/82491180-50e0fe00-9ab2-11ea-98cb-9f6fa905e5ff.PNG)|

Reply:
Nice catch! didn't see that. I will definitely fix that. Thanks!